pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedPrivateFieldRule.java

```diff
@@ -137,26 +130,4 @@ private boolean actuallyUsed(List<NameOccurrence> usages) {
     private boolean isOK(String image) {
         return "serialVersionUID".equals(image) || "serialPersistentFields".equals(image) || "IDENT".equals(image);
     }
-
-    /**
-     * Checks whether the given node is annotated with annotation in the set.
-     * The node should be variable Name declaration.
-     *
-     * @param node
-     *            the node to check
-     * @return <code>true</code> if the annotation has been found
-     */
-    protected boolean hasNeglectAnnotation(VariableNameDeclaration node) {
```

Comment:
removing this would be a breaking API change. We can deprecate it and stop using it internally though

Reply:
Done.